apps/backoffice-v2/src/lib/blocks/hooks/useDocumentBlocks/useDocumentBlocks.tsx

```
@@ -409,6 +409,7 @@ export const useDocumentBlocks = ({
                         }
 
                         if (ocrResult?.parsedData?.[title]) {
+                          debugger;
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)